tests/acceptance/features/apiSpacesShares/shareSpaces.feature

```diff
@@ -239,14 +288,51 @@ Feature: Share spaces
   Scenario Outline: User increases permissions for one member of the group or for the entire group
     Given group "sales" has been created
     And the administrator has added a user "Brian" to the group "sales" using GraphApi
-    And user "Alice" has shared a space "share space" to <firstRecipient> with role "viewer"
+    And user "Alice" has shared a space "share space" with settings:
+      | shareWith | <firstRecipient> |
+      | shareType | <firstShareType> |
+      | role      | viewer           |
     When user "Brian" uploads a file inside space "share space" with content "Test" to "test.txt" using the WebDAV API
     Then the HTTP status code should be "403"
-    When user "Alice" shares a space "share space" to <secondRecipient> with role "editor"
+    When user "Alice" has shared a space "share space" with settings:
+      | shareWith | <secondRecipient> |
+      | shareType | <secondShareType> |
+      | role      | editor            |
     Then the HTTP status code should be "200"
     When user "Brian" uploads a file inside space "share space" with content "Test" to "test.txt" using the WebDAV API
     Then the HTTP status code should be "201"
     Examples:
-      | firstRecipient | secondRecipient |
-      | group "sales"  | user "Brian"    |
-      | user "Brian"   | group "sales"   |
+      | firstRecipient | firstShareType | secondRecipient | secondShareType |
+      | sales          | 8              | Brian           | 7               |
+      | Brian          | 7              | sales           | 8               |
+
+
+  Scenario Outline: A Space Admin can share a space to the user with an expiration date
+    When user "Alice" shares a space "share space" with settings:
+      | shareWith  | Brian                    |
+      | role       | <role>                   |
+      | expireDate | 2042-03-25T23:59:59+0100 |
+    Then the HTTP status code should be "200"
+    And the user "Brian" should have a space called "share space" granted to user "Brian" with role "<role>" and expiration date "2042-03-25"
```

Comment:
so the time and the timezone information gets removed from the send request
what happens if we send only `2042-03-25`?

Reply:
we get 400 with `<message>could not parse expireDate</message>` https://github.com/owncloud/ocis/issues/5452

I send `2042-03-25T23:59:59+0100` in the request but check only `2042-03-25` in the response